packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -1986,3 +2117,199 @@ String _concatStrings({
     return nestedLabel;
   return '$thisString\n$nestedLabel';
 }
+
+/// Provides a way to specify the order in which semantic nodes are sorted.
```

Comment:
+1 to the comprehensive documentation with example.

Reply:
Thanks!